app/src/main/res/layout/activity_authentication.xml

```diff
@@ -7,7 +7,7 @@
     android:layout_height="match_parent">
 
     <ImageView
-        android:id="@+id/imageView"
+        android:id="@+id/profileIcone"

```

Comment:
why has this been changed here ? seems like nothing to do with your profileFragment

Reply:
It does though: it's on this icon you have to click to launch the profileFragment.